pkg/antctl/antctl.go

```diff
@@ -161,6 +163,38 @@ var CommandList = &commandList{
 			commandGroup:        flat,
 			transformedResponse: reflect.TypeOf(agentinfo.AntreaAgentInfoResponse{}),
 		},
+		{
+			use:   "pod-interface",
+			short: "Print Pod's basic interface information",
+			long:  "Print information about the network interface(s) created by the Antrea agent for the specified Pod.",
+			agentEndpoint: &endpoint{
+				nonResourceEndpoint: &nonResourceEndpoint{
+					path: "/podinterface",
+					params: []flagInfo{
+						{
+							name:  "name",
+							usage: "Retrieve pod interface by name",
```

Comment:
Didn't we decide to use "Pod" consistently across all user-facing messages?

Reply:
Also please indicate that a name should not be provided if `--all-namespaces` is used.

Reply:
Did we? This command only shows pod interface related info...

Reply:
I was just pointing out that sometimes you use "pod" in messages and sometimes "Pod". I think we decided in the past to capitalize K8s resource type names. This may or may not have been the right decision :). It seems that K8s itself capitalizes resource type names in the documentation, but not in `kubectl`...

Reply:
Thanks for reminding that I didn't keep them consistent. Since decided in the past, then let's  get them capitalized.